core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/store/dot-experiments-configuration-store.ts

```diff
@@ -752,7 +764,8 @@ export class DotExperimentsConfigurationStore extends ComponentStore<DotExperime
         private readonly dotMessageService: DotMessageService,
         private readonly dotHttpErrorManagerService: DotHttpErrorManagerService,
         private readonly messageService: MessageService,
-        private readonly title: Title
+        private readonly title: Title,
+        private readonly route: ActivatedRoute
     ) {
         super(initialState);
```

Comment:
Here get the data from the resolver and save it in the store.

Reply:
done